tools/Uno.Sdk.Updater/Uno.Sdk.Updater.csproj

```diff
@@ -15,7 +15,7 @@
 
   <ItemGroup>
     <PackageReference Include="NuGet.Packaging" Version="6.10.0" />
-    <PackageReference Include="GitVersion.MsBuild" Version="5.12.0">
+    <PackageReference Include="GitVersion.MsBuild" Version="5.10.3">
```

Comment:
Wondering why the downgrade here

Reply:
1) 5.12.0 was generating the wrong version
2) 5.10.3 matches the version we currently use in the CI build for Azure DevOps, and generates the correct version

Reply:
Thanks for the info @dansiegel